src/main/java/kr/hhplus/be/server/concert/adapter/in/event/PaymentSuccessConsumer.java

```diff
@@ -0,0 +1,76 @@
+package kr.hhplus.be.server.concert.adapter.in.event;
+
+import kr.hhplus.be.server.common.exception.CustomException;
+import kr.hhplus.be.server.common.exception.ErrorCode;
+import kr.hhplus.be.server.concert.domain.ConcertDate;
+import kr.hhplus.be.server.concert.domain.Seat;
+import kr.hhplus.be.server.concert.domain.enums.SeatStatus;
+import kr.hhplus.be.server.concert.port.out.ConcertDateRepository;
+import kr.hhplus.be.server.concert.port.out.SeatRepository;
+import kr.hhplus.be.server.concert.usecase.ConcertSoldOutManager;
+import kr.hhplus.be.server.payment.domain.PaymentSuccessEvent;
+import kr.hhplus.be.server.queue.port.out.QueueTokenRepository;
+import kr.hhplus.be.server.queue.usecase.QueueService;
+import kr.hhplus.be.server.reservation.port.out.SeatHoldRepository;
+import kr.hhplus.be.server.reservation.usecase.ReservationService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.UUID;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class PaymentSuccessConsumer {
+
+    private final ConcertDateRepository concertDateRepository;
+    private final SeatRepository seatRepository;
+    private final SeatHoldRepository seatHoldRepository;
+    private final QueueTokenRepository queueTokenRepository;
+    private final ConcertSoldOutManager concertSoldOutManager;
+    private final ReservationService reservationService;
+    private final QueueService queueService;
+
+    /**
+     * 결제 성공시 이벤트 수신
+     * 1. 해당 콘서트 매진 체크
+     * 2. 매진시 랭킹 업데이트
+     * @param event 결제 성공 이벤트 정보
+     */
+    @KafkaListener(topics = "payment.success", groupId = "${spring.kafka.consumer.group-id}")
```

Comment:
```suggestion
    @KafkaListener(topics = "payment.success", groupId = "${kafka.consumer.group-id.payement-success}")
```
지금은 모든 서비스가 하나의 컨슈머 그룹을 사용하고 있는데요 
MSA 로 전환하게 된다면 컨슈머 그룹도 각각 나눠지면 좋겠네요 ㅎㅎ

Reply:
피드백 덕분에 놓칠 수 있는 부분을 점검할 수 있었습니다. 
현재는 각 컨슈머가 독립된 groupId를 사용하도록 수정했습니다.
정말 감사합니다! 🙏